sources/US/UT/blackbear.json

```diff
@@ -1,15 +1,17 @@
 {
-   "name": "UT Black Bear units",
-   "url": "http://dwrcdc.nr.utah.gov/ucdc/DownloadGIS/Data/Abiotic/2016_Black_Bear.zip",
+   "name": "UT Black Bear Hunt Units",
+   "url": "https://dwrcdc.nr.utah.gov/ucdc/DownloadGIS/Data/Abiotic/2017_Black_Bear.zip",
```

Comment:
This data looks like it is versions yearly, so the thing to do is leave the existing file, and make a new one for 2017 with 2017 in the filename, and put year in the name.